test/test_autograd.py

```diff
@@ -7595,6 +7595,7 @@ def fn(v):
             nnz = 0 if empty_nnz else 5
             _test(sparse_size + dense_size, len(sparse_size), nnz, device)
 
+    @skipMeta
```

Comment:
What causes this to break?

Reply:
In fact, there are many more tests that I'm adding `@skipMeta`. That's mainly because of `torch.sparse_coo_tensor`. Since both `max` and `min` are now structured, it ends up bumping into some error when trying to get an accessor for these meta tensors. It fails here:

https://github.com/pytorch/pytorch/blob/2e49c5dc37bce5ce8ba463a59aacb5d3e4a638b6/c10/core/TensorImpl.h#L1094-L1098

Reply:
Cool!

I stared at the `torch.sparse_coo_tensor()` code a bit - I think we hit issues because part of the logic for validating a sparse tensor is to make sure that all of the indices are in bounds, which requires reading the data.

I guess there's no reason why we'd expect sparse meta to work out of the box - and it probably needs some design to figure out what that interaction should look like. So seems reasonable to skip these for now.